api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -91,10 +92,20 @@ public class FireCloudServiceImpl implements FireCloudService {
   private final Provider<WorkspacesApi> endUserLenientTimeoutWorkspacesApiProvider;
   private final Provider<WorkspacesApi> serviceAccountWorkspaceApiProvider;
 
-  private final FirecloudRetryHandler retryHandler;
-  private final RawlsRetryHandler rawlsRetryHandler;
+  // old Terms of Service endpoints, before Nov 2023 update
+  @Deprecated
+  private final Provider<org.pmiops.workbench.firecloud.api.TermsOfServiceApi>
```

Comment:
We reference two `TermsOfServiceApi` classes!  One is a Firecloud-Orchestration client (this line) and one is a Sam client.  This PR migrates partway to Sam, and we expect to complete that migration as part of this epic.